tests/Feature/FunctionAggregatesMetadataTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+$keyspace = 'functions_aggregates_metadata';
+
+//beforeAll(function () use ($keyspace) {

```

Comment:
Why is this code snippet commenting?

If it doesn't make sense to be tested or implemented, you could delete it from the codebase.

Reply:
This code is commented out, because UDF (User Defined Functions) are considered experimental in ScyllaDB, they work in Apache Cassandra and ScyllaDB(but only if we enable experimental features).
These tests were left here as a placeholder when we setup Apache Cassandra cluster to test against it also

Reply:
Got it, thanks for the explanation!